README.rst

```diff
@@ -165,6 +176,34 @@ Execute command
         avhclient -b aws --ami-version 1.1.0 --specfile AVH-GetStarted/basic/avh.yml (Create a new AVH instance from a v1.1.0 AVH AMI)
         avhclient -b aws --ami-version >1.1.0 --specfile AVH-GetStarted/basic/avh.yml (Create a new AVH instance from a >v1.1.0 AVH AMI)
 
+Execute with Docker
+###################
+
+To run avhclient in a Docker container one needs to create an environment file
+(``env.txt``) with the following content::
+
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_IAM_PROFILE
+    AWS_SECURITY_GROUP_ID
+    AWS_SUBNET_ID
+    AWS_S3_BUCKET_NAME
+    AWS_DEFAULT_REGION
+    AWS_AMI_ID
+    AWS_AMI_VERSION
+    AWS_KEEP_EC2_INSTANCES
+    AWS_KEY_NAME
+    AWS_INSTANCE_TYPE
+    AWS_INSTANCE_NAME
+
+This environment file is used to forward the local environment variables into
+the Docker container. Having this prepared one can run ``avhclient`` in a
+container as follows::
+
+    docker run --rm -i --env-file ./env.txt -v $(pwd):/workspace -w /workspace ghcr.io/arm-software/avhclient avhclient [..]
```

Comment:
If you can add the ~/.aws example here would be great.

Reply:
Here is a powershell and bash compatible volume map for AWS creds and ssh keys:

`-v $HOME/.ssh:/root/.ssh -v $HOME/.aws:/root/.aws`